{"name": "test_register_valid_new_user", "status": "failed", "statusDetails": {"message": "AssertionError: Fails\nassert 1 == 2", "trace": "self = <test_register_new_user.TestRegisterNewUser object at 0x000001DB9D9F3820>\n\n    @pytest.mark.TC02\n    def test_register_valid_new_user(self):\n    \n        my_account_signout =MyAccountSignedOut(self.driver)\n        my_account_signin=MyAccountSignedIn(self.driver)\n    \n        rand_email_and_pass=generate_random_email_and_password()\n        #go to my account\n        my_account_signout.go_to_account_page()\n        my_account_signout.register_email_input(rand_email_and_pass['email'])\n>       assert 1 == 2, 'Fails'\nE       AssertionError: Fails\nE       assert 1 == 2\n\nSSqatest\\tests\\my_account\\test_register_new_user.py:20: AssertionError"}, "attachments": [{"name": "stdout", "source": "e49c04e2-fc43-4657-bb48-502a2679ba7a-attachment.txt", "type": "text/plain"}], "start": 1643299810079, "stop": 1643299816011, "uuid": "65b60bed-cee8-444b-96bf-289b43c343a5", "historyId": "4a4514772bd99a6d1ef200c403f6e98c", "testCaseId": "aecf929ef9bc80c3b76b13263e946cb7", "fullName": "SSqatest.tests.my_account.test_register_new_user.TestRegisterNewUser#test_register_valid_new_user", "labels": [{"name": "tag", "value": "TC02"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "SSqatest.tests.my_account"}, {"name": "suite", "value": "test_register_new_user"}, {"name": "subSuite", "value": "TestRegisterNewUser"}, {"name": "host", "value": "DESKTOP-3Q5JSNM"}, {"name": "thread", "value": "12648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "SSqatest.tests.my_account.test_register_new_user"}]}